docs/deployment/kubernetes.md

```diff
@@ -205,12 +202,17 @@ If you are running into any issues please check out the troubleshooting guide an
 
 This section covers additional advanced topics beyond the initial deployment.
 
+#### Deploy with SSL
+To enable SSL while using Helm, try one of the following references:
+
+- [Using nginx-ingress and cert-manager](/reference/ssl/kubernetes-with-cert-manager.md)
+
 #### Use a private registry with Kubernetes
 
 If you are using a hosted private Docker registry ([Docker Hub](https://hub.docker.com/), or other),
 in order to check how to configure it, please visit the Kubernetes [documentation](https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry).
 
-If you try to deploy using `faas-cli deploy` it will fail because the Kubernetes kubelet component will not have credentials to authorize the pull request. 
+If you try to deploy using `faas-cli deploy` it will fail because the Kubernetes kubelet component will not have credentials to authorize the pull request.
```

Comment:
Do we mean `pull request` - pull request is a loaded term. Perhaps that could be edited?